examples/ingest-file.ts

```diff
@@ -1,14 +1,19 @@
 import fs from 'fs';
 import Client, { CloudURL } from '../lib';
-import { ContentEncoding, ContentType } from '../lib/datasets';
+import { datasets } from '../lib/datasets';
```

Comment:
Would it be better to import the namespace from `../lib` directly as that's how it's used from external packages?